Vagrantfile

```diff
@@ -92,8 +122,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   (0..NRGWS - 1).each do |i|
     config.vm.define "rgw#{i}" do |rgw|
       rgw.vm.hostname = "ceph-rgw#{i}"
-      rgw.vm.network :private_network, ip: "#{SUBNET}.4#{i}"
-
+      if !OSVM
+        rgw.vm.network :private_network, ip: "#{SUBNET}.4#{i}"
```

Comment:
can you rebase your patch again? so you get this fix: https://github.com/ceph/ceph-ansible/pull/535